bag_verifier/src/main/scala/weco/storage_service/bag_verifier/fixity/bag/BagExpectedFixity.scala

```diff
@@ -55,29 +54,25 @@ class BagExpectedFixity[BagLocation <: Location, BagPrefix <: Prefix[
     matched: MatchedLocation
   ): Either[Throwable, ExpectedFileFixity] =
     matched match {
-      case MatchedLocation(
-          bagPath: BagPath,
-          checksum: Checksum,
-          Some(fetchEntry)
-          ) =>
+      case loc @ MatchedLocation(bagPath, _, _, Some(fetchEntry)) =>
```

Comment:
In particular, here I’m going to use the multiChecksum from this unpacking and store that in the *FileFixity case classes, and then in the FixityChecker I'll be comparing that to the multiChecksum that we read.